_posts/2015-01-22-release-notes.md

```diff
@@ -5,9 +5,15 @@ category: ref
 date: 2015-03-04 16:49:01
 ---
 
-### Current release: ***beta5***
-### PoyntOS SDK Version: 1.2.11
-### Poynt API Model Version: 1.2.44
+### Current release: ***beta7***
+### PoyntOS SDK Version: 1.2.14
+### Poynt API Model Version: 1.2.62
+
+### Changes since beta6 (1.2.11)
+* Updated API Model with new modifiers support (`selectableVariations`)
```

Comment:
should be more specific -changes in both product and order api models. also call out the backwards incompatibilities.